bootloader/main.c

```diff
@@ -408,20 +437,39 @@ get_real_prog_path(void)
     return result;
 }
 
+char *get_tmp_root(void)
+{
+    char *root_dir;
+    root_dir = TMPROOTDIR;
+
+    char *ret;
+	ret = (char*) malloc (strlen(root_dir)+1);
+	strcpy(ret, root_dir);
+    ret = strtrim(ret);
```

Comment:
Why would we need to trim whitespace? Certainly staticx should be capable of writing a proper NUL-terminate C string.